docs/OVERVIEW.md

```diff
@@ -0,0 +1,348 @@
+# BioIO
+
+[![Build Status](https://github.com/bioio-devs/bioio/actions/workflows/ci.yml/badge.svg)](https://github.com/bioio-devs/bioio/actions)
+[![Documentation](https://github.com/bioio-devs/bioio/actions/workflows/docs.yml/badge.svg)](https://bioio-devs.github.io/bioio)
```

Comment:
I don't know if we need the documentation and build status badges here in this doc.